v16.4 requires direct dependency on `tslib`?

I am trying to upgrade prettier-eslint from 16.3.2 to 16.4.1.  I am get the following error:

```
Error: Cannot find module 'tslib'
Require stack:
- /project/ui-bundle/node_modules/prettier-eslint/lib/index.js
- /project/ui-bundle/node_modules/prettier-eslint/index.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)
    at Function._load (node:internal/modules/cjs/loader:1075:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:219:24)
    at Module.require (node:internal/modules/cjs/loader:1340:12)
    at require (node:internal/modules/helpers:138:16)
    at Object.<anonymous> (/project/ui-bundle/node_modules/prettier-eslint/lib/index.js:5:17)
    at Module._compile (node:internal/modules/cjs/loader:1565:14)
    at Object..js (node:internal/modules/cjs/loader:1708:10)
    at Module.load (node:internal/modules/cjs/loader:1318:32)
    at Function._load (node:internal/modules/cjs/loader:1128:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:219:24)
    at Module.require (node:internal/modules/cjs/loader:1340:12)
    at require (node:internal/modules/helpers:138:16)
    at Object.<anonymous> (/project/ui-bundle/node_modules/prettier-eslint/index.js:1:29)
    at Module._compile (node:internal/modules/cjs/loader:1565:14)
    at Object..js (node:internal/modules/cjs/loader:1708:10)
    at Module.load (node:internal/modules/cjs/loader:1318:32)
    at Function._load (node:internal/modules/cjs/loader:1128:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:219:24)
    at cjsLoader (node:internal/modules/esm/translators:263:5)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:196:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:271:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:547:26) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/project/ui-bundle/node_modules/prettier-eslint/lib/index.js',
    '/project/ui-bundle/node_modules/prettier-eslint/index.js'
  ]
}
```


I can resolve this error if I add a dependency on `tslib` to my `package.json`, but is this expected that projects using this must now have a dependency on `tslib` even if they are not using TypeScript?

--- a/.changeset/ready-mangos-run.md
+++ b/.changeset/ready-mangos-run.md
@@ -1,0 +1,5 @@
+---
+"prettier-eslint": patch
+---
+
+fix: `tslib` should be listed as `dependency`

--- a/package.json
+++ b/package.json
@@ -62,6 +62,7 @@
     "prettier": "^3.5.3",
     "pretty-format": "^29.7.0",
     "require-relative": "^0.8.7",
+    "tslib": "^2.8.1",
     "vue-eslint-parser": "^9.4.3"
   },
   "devDependencies": {
@@ -99,7 +100,6 @@
     "strip-indent": "^3.0.0",
     "svelte": "^4.2.19",
     "svelte-eslint-parser": "^0.33.1",
-    "tslib": "^2.8.1",
     "typescript": "^5.8.3"
   },
   "resolutions": {

